pkg/tcpip/header/ndp_options.go

```diff
@@ -511,6 +521,10 @@ func (o NDPPrefixInformation) PrefixLength() uint8 {
 	return o[ndpPrefixInformationPrefixLengthOffset]
 }
 
+func (o NDPPrefixInformation) SetPrefixLength(len uint8) {
```

Comment:
Please add unit tests for each of the setters created in this change :) 

Also, please add comments for each of the exported functions:
```
// FUNCTION_NAME does X, Y and Z.
```